docs/development.rst

```diff
@@ -627,14 +627,8 @@ Code style
 
 C code is formatted using
 `clang-format <https://clang.llvm.org/docs/ClangFormat.html>`_
-with a custom configuration.
-To ensure that your code is correctly formatted, you can run
-
-.. code-block:: bash
-
-    clang-format -i c/tskit/* c/tests/*.c c/tests/*.h
-
-before submitting a pull request.
```

Comment:
I think it's nice to leave in the DIY command? pre-commit checks a lot of things and can produce a lot of output, so it's less overwhelming to be able to easily run them separately sometimes.  Unless this is out of date somehow?

Reply:
It could say "If you have clang-format installed, you can run the check manually as follows..."

Reply:
I agree with Peter. But just noticed that when you run clang-format in `c/tskit/trees.c` it changes sth that it shouldnt.

Reply:
>  But just noticed that when you run clang-format in c/tskit/trees.c it changes sth that it shouldnt.
What version of `clang-format`? It is not stable across releases. CI and this pre-commit use 6.0.

Reply:
hah, thanks! I had 5.0.1

Reply:
Oh, well if it depends precisely on the version, maybe we should just say "You can run clang-format yourself, but note that the results depend on clang-format's version, and we use a specific version in the pre-commit checks."

Reply:
Or even "We don't recommend you run clang-format yourself, because the results depends on clang-format's version."

Reply:
I've re-added a manual running snippet and added a note about the version.